code/modules/mob/living/carbon/resist.dm

```diff
@@ -200,12 +199,13 @@
 	return ..()
 
 /mob/living/carbon/escape_buckle()
-	setClickCooldown(100)

```

Comment:
I've been looking for this bastard for a long while, and it's been right at the start of the proc this whole time.

Bless your soul

Reply:
Second thought - might be good to make it add a cooldown if cuffed so chatspam doesn't happen.

Reply:
What would be chatspammed?

Reply:
```
		visible_message(
			"<span class='danger'>[usr] attempts to unbuckle themself!</span>",
			"<span class='warning'>You attempt to unbuckle yourself. (This will take around 2 minutes and you need to stand still)</span>"
			)
```
That would, if the person has cuffs on. Unless I'm retarded.

Reply:
popping the cooldown right before it would probably work fine

Reply:
It's already before the cooldown.

Reply:
I, instead of saying I was wrong. will opt to say I'm just retarded, instead.

Reply:
Also it wouldn't have mattered where the cooldown was set, as the cooldown check is checked before the proc is called.